life_expectancy/tests/test_cleaning.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for the cleaning module"""
+from unittest.mock import patch, Mock
+import pytest
+import pandas as pd
+from life_expectancy.cleaning import load_data, clean_data, save_data
+from life_expectancy.tests.fixtures.mock_date import data_raw, data_expect
+from . import OUTPUT_DIR
+
+@pytest.fixture
+def fixture_raw():
+    """
+    Load mock raw data
+    :return: Dict with raw data mocked
+    """
+
+    return pd.DataFrame(data_raw())
+
+@pytest.fixture
+def fixture_expect():
+    """
+    Load mock expected data
+    :return: Dict with prepared data mocked
+    """
+
+    return pd.DataFrame(data_expect())
+
+@patch("pandas.read_table")
```

Comment:
```suggestion
@patch("life_expectancy.cleaning.pd.read_table")
```

Reply:
I did some tests, with "pandas.read_table" and with "life_expectancy.cleaning.pd.read_table" it worked fine, but with "life_expectancy.cleaning.pandas.read_table" failed, so I changed to your suggestion